.github/actions/utility-action/action.yml

```diff
@@ -0,0 +1,112 @@
+name: Utility Actions
+
+description: |
+  A set of utility steps to be used across different workflows, including:
+  - Installing Terraform
+  - Installing AWS CLI
+  - Setting Terraform variables
+  - Checking/Creating an S3 bucket
+
+inputs:
+  awscli-version:
+    description: 'Version of the AWS CLI to install'
+    required: true
+    # renovate: datasource=github-releases depName=aws/aws-cli
+    default: '2.15.52'
```

Comment:
just a suggestion from my side. If you anyway define and only need it in the utility action, why not try to reduce the amount of times you're suggesting the value for aws cli?
Instead just call it `latest`, add some logic here that if `latest` use the pinned version`, otherwise whatever was input.
As every other action was defining the value as well + renovate.

Reply:
I see what you mean. In that case, let's maybe just keep it as is. It's automated and Renovate will deal with it.